x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_logic.ts

```diff
@@ -109,14 +110,13 @@ export const CredentialsLogic = kea<CredentialsLogicType>({
           ...apiTokens.filter((token) => token.name !== apiToken.name),
           apiToken,
         ],
-        onApiKeyDelete: (apiTokens, tokenName) =>
-          apiTokens.filter((token) => token.name !== tokenName),
       },
     ],
     meta: [
-      {},
+      DEFAULT_META,
```

Comment:
Changed from `{}` to `DEFAULT_META` per https://github.com/elastic/kibana/pull/91565#discussion_r577152950. This accomplishes a few things:

1. `10` (over the current default 25 from the API) is generally a more friendly # of items to parse through UI-wise, especially for smaller screens
2. Gives us more standard pagination across App Search
3. This potentially allows us to someday easily remove `hidePerPageOptions: true` (which allows users to customize the # of items per page to their preference), if either our users or product request the functionality:
<img width="180" alt="" src="https://user-images.githubusercontent.com/549407/108776206-f2502100-7516-11eb-8e94-ba9b76ce325c.png">